### PR TITLE
Update 'Alpha Exploration Co., Inc' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.netlify.com/#!doc=%7B%22slug%22%3A%22joinclubhouse%22%2C%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22social%20media%22%5D%2C%22name%22%3A%22Alpha%20Exploration%20Co.%2C%20Inc%22%2C%22runs%22%3A%5B%22Clubhouse%22%5D%2C%22address%22%3A%22Alpha%20Exploration%20Co.%5Cn548%20Market%20St%5CnPMB%2072878%5CnSan%20Francisco%2C%20California%2094104-5401%22%2C%22phone%22%3A%22415-549-1842%22%2C%22email%22%3A%22support%40alphaexplorationco.com%22%2C%22web%22%3A%22https%3A%2F%2Fwww.joinclubhouse.com%2F%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.notion.so%2FPrivacy-Policy-cd4b415950204a46819478b31f6ce14f%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F934%22%2C%22https%3A%2F%2Fbusinesssearch.sos.ca.gov%2FDocument%2FRetrievePDF%3FId%3D04563531-27853656%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F937%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F995%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/joinclubhouse.json [hunk 1]
```diff
@@ -10,14 +10,16 @@
     "runs": [
         "Clubhouse"
     ],
-    "address": "5335 Lawton Avenue\nOakland\nCalifornia 94618\nUnited States of America",
+    "address": "Alpha Exploration Co.\n548 Market St\nPMB 72878\nSan Francisco, California 94104-5401",
```
Comment (fm) [inline companies/joinclubhouse.json:13]: Can't find any sources for the new address. I've emailed them to confirm the correct address.

File: companies/joinclubhouse.json [hunk 2]
```diff
@@ -10,14 +10,16 @@
...
+    "phone": "415-549-1842",
```
Comment (fm) [inline companies/joinclubhouse.json:14]: Not formatted correctly, `+1 415 549 1842`

**Review by fm — CHANGES_REQUESTED** [review]

File: companies/joinclubhouse.json (hunk 1 above)
Comment (fm) [reply companies/joinclubhouse.json:13]: @mal-tee Never got an answer from them, but California Secretary of State documents were filed with the change of address, we're good to merge.

Can we add the doc as a source too? 
https://businesssearch.sos.ca.gov/Document/RetrievePDF?Id=04563531-29903880

File: companies/joinclubhouse.json (hunk 2 above)
Comment (mal-tee) [reply companies/joinclubhouse.json]: Do we have a source for this number? 🤔

File: companies/joinclubhouse.json (hunk 1 above)
Comment (baltpeter) [reply companies/joinclubhouse.json]: > Can we replace the current link with this one, as well?
> [businesssearch.sos.ca.gov/Document/RetrievePDF?Id=04563531-29903880](https://businesssearch.sos.ca.gov/Document/RetrievePDF?Id=04563531-29903880)

That's an interesting point that we currently don't have guidelines on. For now, I'll replace the old source but I have opened #1042 to discuss this for the future.

Comment (mal-tee) [reply companies/joinclubhouse.json]: I already did that :P

**Review by baltpeter — APPROVED** [review]
Thank you!

File: companies/joinclubhouse.json (hunk 2 above)
Comment (fm) [reply companies/joinclubhouse.json]: Negative, I think we should remove it. I doubt they would answer any of the phones listed anyway. 

I also found [this source](https://www.dnb.com/business-directory/company-profiles.alpha_exploration_co.2e9f2a57c1f5b0aebc20563d279ad536.html) with a different number.

Comment (mal-tee) [reply companies/joinclubhouse.json]: Thanks! I agree.

**Review by mal-tee — APPROVED** [review]

**Review by baltpeter — APPROVED** [review]

**Pull request merged**